src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -232,23 +232,21 @@ def tokenize_card(self, order=None, billing_address=None, user=None):
             return self._tokenize_card_for_user(user)
         return self._tokenize_card_for_order(order, billing_address)
 
-    def create_payment(self, order, billing_address, installment=None):
+    def create_payment(self, order, installment, billing_address):
         """
         Create a payment object for a given order
 
         https://docs.lyra.com/fr/rest/V4.0/api/kb/token_with_payment.html
         https://docs.lyra.com/fr/rest/V4.0/api/playground/Charge/CreatePayment
         """
         url = f"{self.api_url}Charge/CreatePayment"
-        payload = self._get_common_payload_data(
-            order, billing_address, installment=installment
-        )
+        payload = self._get_common_payload_data(order, installment, billing_address)
         payload["formAction"] = "REGISTER_PAY"
 
         return self._get_payment_info(url, payload)
 
     def create_one_click_payment(
-        self, order, billing_address, credit_card_token, installment=None
+        self, order, billing_address, installment, credit_card_token=None
     ):
```

Comment:
Why the `credit_card_token` is optional now ?

Reply:
Just a mistake…